ui/src/theme.js

```diff
@@ -11,31 +11,36 @@ const SPACING = {
 export default createTheme({
   palette: {
     primary: {
-      light: '#804cc5',
-      main: '#4e1f94',
-      dark: '#160065',
+      main: '#000',
     },
     secondary: {
-      light: '#678dff',
-      main: '#0061f2',
-      dark: '#0038be',
+      main: '#c50042',
     },
-    error: red,
```

Comment:
Should these be updated? What changes when you delete these?

Reply:
When I delete them the theme changes to the original theme it was. for example the dashboard becomes blue when I delete the palette

Reply:
Oh OK I see. Can we pick similar colors from the protocol pallette?

Reply:
Yes we do have similar colours in the protocol pallette, we can pick them if they look more lively. Should I update the pallette to include those colours?